app/lib/frontend/templates/views/pkg/install_tab.mustache

```diff
@@ -19,11 +19,11 @@
 <p>Run this command:</p>
 {{#use_pub_get}}
 <p>With Dart:</p>
-<pre><code class="language-shell"> $ dart pub add {{ package }}</code></pre>
+<pre data-text-to-copy="dart pub add {{ package }}"><code class="language-shell"> $ dart pub add {{ package }}</code></pre>
 {{/use_pub_get}}
 {{#use_flutter_packages_get}}
 <p>With Flutter:</p>
-<pre><code class="language-shell"> $ flutter pub pub add {{ package }}</code></pre>
+<pre data-text-to-copy="flutter pub pub add {{ package }}"><code class="language-shell"> $ flutter pub pub add {{ package }}</code></pre>
```

Comment:
`fluttee pub add` should work now... No need for pub pub

Reply:
Updated.